app/helpers/arclight_helper.rb

```diff
@@ -43,6 +43,17 @@ def fields_have_content?(document, field_accessor)
     end
   end
 
+  # If we have a facet on the repository, then return the Repository object for it
+  #
+  # @return [Repository]
+  def repository_faceted_on
+    if try(:search_state) # rubocop: disable Style/GuardClause
```

Comment:
If we want to follow the style guide w/o doing an inline disable here, we could add `return unless try(:search_state)` at the top of the method (which should be functionally equivalent).